app/templates/items/detail.hbs

```diff
@@ -1,13 +1,13 @@
 <section class="main-section item_details_screen">
   <div class="search">
     <div class="row profile_container">
-      {{#link-to "items.index" tagName='span' href=false}}
+      {{#back-link}}
```

Comment:
I think this will create the item detail tabs back and forth issue just like order tabs. Because AFAIK, back-link is nothing but `window.history.back()`.

Reply:
There's always potential issues, but it's the behavior we want eventually. We'll fix the place it's navigating from later on if it does :)